actors/miner/tests/prove_commit_niporep.rs

```diff
@@ -0,0 +1,141 @@
+use fvm_ipld_encoding::ipld_block::IpldBlock;
+use fvm_shared::{clock::ChainEpoch, error::ExitCode};
+
+use fil_actor_miner::{Actor, Method};
+use util::*;
+
+mod util;
+
+const PERIOD_OFFSET: ChainEpoch = 100;
+
+#[test]
+fn prove_single_sector_ni_fail() {
+    let h = ActorHarness::new(PERIOD_OFFSET);
+    let rt = h.new_runtime();
+    rt.balance.replace(BIG_BALANCE.clone());
+    let miner = rt.receiver.id().unwrap();
+
+    let seal_randomness_epoch = PERIOD_OFFSET + 1;
+    let expiration = seal_randomness_epoch + 1000;
+
+    rt.set_epoch(seal_randomness_epoch);
+    h.construct_and_verify(&rt);
+
+    rt.set_epoch(seal_randomness_epoch + 400);
+
+    let params = h.make_prove_commit_ni_params(miner, &[100], seal_randomness_epoch, expiration, 0);
+
+    let res = rt.call::<Actor>(
+        Method::ProveCommitSectorsNI as u64,
+        IpldBlock::serialize_cbor(&params).unwrap(),
+    );
+
+    assert!(res.is_err());
+    assert_eq!(res.unwrap_err().exit_code(), ExitCode::USR_ILLEGAL_ARGUMENT);
+}
+
+#[test]
+fn prove_two_sectors_aggregate_ni() {
```

Comment:
these tests only really assert on the deadline, but you could also be checking the SectorOnChainInfo and even whether they're now enrolled in cron (I think)

Reply:
I think this was an oversight in the FIP because it was a modified merge with 0090 and the original 0092 which was just focused on user-specified partitions so the only test cases listed in there are about partitions but that seems a little deficient to me.

Reply:
Added `SectorOnChainInfo` check. Any example how to test if sectors are enrolled in cron?

Reply:
It might be good enough to just check `deadline_cron_active` on the miner actor's state; that doesn't tell us about cron but tbh I don't know how to do that and it does sound overly complex!